app/backend/src/couchers/migrations/env.py

```diff
@@ -1,49 +1,54 @@
 from logging.config import fileConfig
+from typing import Any
 
 from alembic import context
+from alembic.config import Config
 from sqlalchemy import engine_from_config, pool
+from sqlalchemy.schema import MetaData
 
 from couchers import models
 from couchers.config import config as couchers_config
 
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
-config = context.config
+config: Config = context.config
 
 config.set_main_option("sqlalchemy.url", couchers_config["DATABASE_CONNECTION_STRING"])
 
 
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
 if config.get_main_option("dont_mess_up_logging", "dont_care") == "dont_care":
+    if not config.config_file_name:
+        raise RuntimeError(config.config_file_name)
```

Comment:
I’m not sure what this does exactly?

Reply:
If for some reason file name is not set, we'll raise an error. It's a sanity check and also makes mypy happy.